posts/20230223/index.qmd

```diff
@@ -0,0 +1,67 @@
+
+By this point, the research question was not sufficiently motivated, and the data was ‘double-dipped’ and overly engineered. In fact, in presenting the comparisons between algorithm performance, the authors showcase how the engineered data performs better at classification of the task, which is of no surprise because the data features were initially chosen because they were better at classifying the task (see paper Fig. 20, 21).
+
+The seeming conclusion the authors draw from this section that XGBoost was a superior classifier, but considering all of these factors above, this conclusion did not follow logic. 
+
+## Alternative approach
+Even if the authors chose not to reconsider their overall approach, wanted to use this binary classification for predicting final sale outcomes as they proposed, and intended to use the same data as was described here for these predictions, there are many changes the authors could have made to improve their modeling methods. I have outlined an alternative binary classification approach the authors could have taken with the existing problem and data framework in the figure below. 
+
+#### Data
+The data should be encoded properly, with categorical data like parid (property identifier), nbhd (neighborhood code), yrblt (year built), zip21 (ZIP code of area), sale_date (sale date), and luc (property class) treated as such. Furthermore, month and year variables should be extracted and matched to the socio-economic data matching it on those variables where possible. 
+
+![Figure 1. Another architecture for how the authors could have approached their high-price/low-price classificaiton problem; compare to the authors' architecture in Figure 9 of the paper.](alternative_approach.png)
+
+#### Model Training
+Before training the model, we should have an understanding about what exactly we want the model to do. Looking at Fig. 2 in the paper, we can see that the occurrence of high price instances surpassed low price instances in the last few years; thus it would probably behoove the authors to build a model that is optimized to predict high-price (Class = 1) outcomes to apply this model to new data. 
```

Comment:
We actually criticized that they framed this as a binary classification. I think a better alternative approach would be using regression.

Reply:
I agree.